OffbeatStore/utility_functions/operations.py

```diff
@@ -244,3 +247,42 @@ def delete_review_operation(id):
     db = get_db()
     db.execute(query, (id,))
     db.commit()
+
+
+def add_notification_operation(user_id, product_id, notification_type):
+    query = """INSERT INTO notification(notifier_id, user_id, product_id, notification_type)
+                VALUES (?, ?, ?, ?)"""
+    db = get_db()
+    try:
+        db.execute(query,
+                   (g.user["id"], user_id, product_id, notification_type)
+                   )
+        db.commit()
+    except:
+        pass
```

Comment:
It is always good to add log message here. Customer will not be affected by the exception but you always want to know when something happen.